library.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "SevenSegmentTM1637",
+  "version": "1.0.0",
```

Comment:
1.0.4, unless you _want_ an older version ;)

Reply:
Right! My bad. This is my first PR